google-cloud-resource_manager/Gemfile

```diff
@@ -4,6 +4,7 @@ gemspec
 
 gem "rake", "~> 11.0"
 gem "google-cloud-core", path: "../google-cloud-core"
+gem "google-cloud-env", path: "../google-cloud-env"
```

Comment:
Because google-cloud-env is a dependency brought in by google-cloud-core, we need to tell Bundler where this library is located. So even though there are no direct uses of it, the dependency still needs to be found. The `Gemfile` does not ship in the gem, it is mostly used for development and testing.